5.5/run-mysqld.sh

```diff
@@ -94,6 +103,29 @@ if [ "$1" = "mysqld" -a ! -d "$MYSQL_DATADIR/mysql" ]; then
 		EOSQL
 	fi
 	mysqladmin $admin_flags flush-privileges shutdown
+}
+
+function generate_config() {
+	# Time zone.
```

Comment:
Default for optioned named "default-time-zone"? There is none. I know because I checked it not in documentation, but by running MySQL and asking with --help --verbose. Doc https://dev.mysql.com/doc/refman/5.5/en/server-options.html#option_mysqld_default-time-zone say that it's from system, usually from env var TZ.